### DISCUSSION TITLE
Delete certificate layout

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Ed/Community#661] [General Discussion]
@vinukumar-vs @rajeevsathish  How can we delete the certificate layout from Sunbird saas version 5.0
If there is any curl for it can you please share

**Comment by vinukumar-vs** [comment]
@siddhishinde0723 There is delete API for certificate layout, but you can delete the cert attached to the batch
<img width="1558" alt="image" src="https://github.com/Sunbird-Ed/Community/assets/4496096/cae11460-2879-4f7e-ba7d-bb5007496de7">

**Comment by anisha147** [comment]
Hi @siddhishinde0723 

We hope the solution shared by Vinu answers your query. If yes, could you please close this thread?

cc: @vinukumar-vs @mohitga
